cloudbuild.yaml

```diff
@@ -47,79 +47,80 @@
 #    https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/go
 
 steps:
+# Blocked by https://github.com/GoogleContainerTools/kaniko/issues/477
 - id: 'Docker Image: open-match-build'
-  name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'open-match-build', '-f', 'Dockerfile.ci', '.']
+  name: gcr.io/kaniko-project/executor
+  args: ['--destination=gcr.io/$PROJECT_ID/open-match-build', '--cache=true', '--cache-ttl=6h', '--dockerfile=Dockerfile.ci', '.']
   waitFor: ['-']
 
 - id: 'Build: Clean'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'clean']
   waitFor: ['Docker Image: open-match-build']
 
 - id: 'Setup: Download Dependencies'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'sync-deps']
   volumes:
   - name: 'go-vol'
     path: '/go'
   waitFor: ['Build: Clean']
 
 - id: 'Build: Install Toolchain'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'install-toolchain']
   volumes:
   - name: 'go-vol'
     path: '/go'
   waitFor: ['Setup: Download Dependencies']
 
 - id: 'Build: Protocol Buffers'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'all-protos']
   volumes:
   - name: 'go-vol'
     path: '/go'
   waitFor: ['Build: Install Toolchain']
 
 - id: 'Lint: Format'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'fmt']
   volumes:
   - name: 'go-vol'
     path: '/go'
   waitFor: ['Build: Protocol Buffers']
 - id: 'Lint: Vet'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'vet']
   volumes:
   - name: 'go-vol'
     path: '/go'
   waitFor: ['Build: Protocol Buffers']
 - id: 'Build: Binaries'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'GOPROXY=off', 'all', '-j8']
   volumes:
   - name: 'go-vol'
     path: '/go'
   waitFor: ['Build: Protocol Buffers']
-- id: 'Build: Test'
-  name: open-match-build
+- id: 'Test: Core'
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'GOPROXY=off', 'test-in-ci']
   volumes:
   - name: 'go-vol'
     path: '/go'
   waitFor: ['Build: Protocol Buffers']
 - id: 'Build: Docker Images'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'VERSION_SUFFIX=$SHORT_SHA', 'build-images', '-j8']
   waitFor: ['Build: Protocol Buffers']
 - id: 'Build: Push Images'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'VERSION_SUFFIX=$SHORT_SHA', 'push-images', '-j8']
   waitFor: ['Build: Docker Images']
 
 - id: 'Build: Generate Kubernetes Deployment Configs'
-  name: open-match-build
+  name: 'gcr.io/$PROJECT_ID/open-match-build'
   args: ['make', 'VERSION_SUFFIX=$SHORT_SHA', 'clean-install-yaml', 'install/yaml/']
   waitFor: ['Build: Install Toolchain']
 
```

Comment:
I can't comment on unchanged section but we should also update the image name for build steps 'Build/Test/Deploy: Website'